BlueShift-iOS-SDK/BlueShiftAppDelegate.h

```diff
@@ -68,7 +68,7 @@
 - (void)registerForSilentPushNotification;
 
 // SceneDelegate lifecycle methods
-- (void)sceneDidBecomeActive:(UIScene* _Nullable)scene API_AVAILABLE(ios(13.0));
+- (void)sceneWillEnterForeground:(UIScene* _Nullable)scene API_AVAILABLE(ios(13.0));
 - (void)sceneDidEnterBackground:(UIScene* _Nullable)scene API_AVAILABLE(ios(13.0));
```

Comment:
should we rename or change this `sceneDidEnterBackground` for compatibility?

Reply:
This method is called before entering the foreground, so it's named as sceneWillEnterForeground. I am using the same name as the OS has.